frameworks/MLNet/exec.py

```diff
@@ -41,6 +41,11 @@ def run(dataset: Dataset, config: TaskConfig):
     os.environ['MODELBUILDER_AUTOML'] = MODELBUILDER_AUTOML
 
     artifacts = config.framework_params.get('_save_artifacts', [])
+    allowedMultiClassifiers = config.framework_params.get('classifiers','LGBM;RF;FASTTREE;LBFGS;SDCA')
+    allowedRegressors = config.framework_params.get('regressors','LGBM;RF;FASTTREE;LBFGS;SDCA')
+    os.environ['AutoMLServiceMultiClassifiers'] = allowedMultiClassifiers
+    os.environ['AutoMLServiceRegressors'] = allowedRegressors
```

Comment:
As these are not purely technical parameters but a way to configure the behaviour of MLNET, we can't allow passing non-default values here.
Also, by convention, parameters that are not passed directly to the training function (in your case, to `run_cmd(cmd)` below) must be prefixed with underscore.

Summing up these 2 concerns, we can only accept something like:
```suggestion
    allowedMultiClassifiers = config.framework_params.get('_classifiers')
    allowedRegressors = config.framework_params.get('regressors')
    if allowedMultiClassifiers is not None:
        os.environ['AutoMLServiceMultiClassifiers'] = allowedMultiClassifiers
    if allowedRegressors is not None:
        os.environ['AutoMLServiceRegressors'] = allowedRegressors
```